client/views/cards/templates.html

```diff
@@ -181,7 +181,9 @@ <h3 class="card-detail-item-header">Labels</h3>
                             <h3 class="card-detail-item-header js-show-with-desc">Description</h3>
                             <a href="#" class="card-detail-item-header-edit hide-on-edit js-show-with-desc js-edit-desc {{ isMemberAll '' 'hide' }}">Edit</a>
                             <div class="current markeddown hide-on-edit js-card-desc js-show-with-desc">
-                                <p>{{ card.description }}</p>
+                              {{#markdown}}
+{{ card.description }}
```

Comment:
I would write it in one single line `{{# markdown }}{{ card.description }}{{/ markdown }}` inside the `<p>` tag.